src/parser/github-comment-module.ts

```
@@ -38,29 +38,61 @@ export class GithubCommentModule extends BaseModule {
     return div.innerHTML;
   }
 
-  async getBodyContent(data: Readonly<IssueActivity>, result: Result, stripContent = false): Promise<string> {
-    const keysToRemove: string[] = [];
+  private _getZeroPriceWarningContent(): string {
+    const warningMessage = `# Rewards Summary\n\n> [!CAUTION]\n> No rewards have been distributed for this task because it was explicitly marked with a Price: 0 label.\n\n`;
```

What you can do that's much simpler is that you can throw an error with a warning message. All you need to do is:

```typescript
throw this.context.logger.warn("No rewards have been distributed for this task because it was explicitly marked with a Price: 0 label.");
```